CMSgov/design-system
# [WNMGDS-2838] Update all themes to make inline tool tips black

## Summary

[Ticket](https://jira.cms.gov/browse/WNMGDS-2838)
Changed the value pointed to in the tokens for '--tooltip-trigger__color` class. 

## How to test

1. Pull down the branch locally and build
2. Confirm that for each theme the color of the tooltip underline and text matches the color of the surrounding text. 

## Verify Screenshots look ok

<img width="300" alt="cms_tooltip" src="https://github.com/user-attachments/assets/08aadb00-7793-4a64-a301-5fe7e956b34d">
<img width="386" alt="cms_tooltip_hover" src="https://github.com/user-attachments/assets/05411876-ac3a-4132-9c5f-d218d49600c3">
<img width="509" alt="core_tooltip" src="https://github.com/user-attachments/assets/857b01c7-9fe0-42db-90b4-0c884b17c56c">
<img width="462" alt="core_tooltip_hover" src="https://github.com/user-attachments/assets/c324d1b4-e182-481c-8d94-a5f21953fa61">
<img width="351" alt="healthcare_tooltip" src="https://github.com/user-attachments/assets/1b92bab3-b568-4459-895b-02eeeb7bcd94">
<img width="377" alt="healthcare_tooltip_hover" src="https://github.com/user-attachments/assets/8142eb57-df9e-41dd-9e06-2577cc691812">
<img width="265" alt="medicare_tooltip" src="https://github.com/user-attachments/assets/48f9296b-8b1c-4de1-b3b1-0f147da3b57e">
<img width="404" alt="medicare_tooltip_hover" src="https://github.com/user-attachments/assets/5c5685f9-d3be-46eb-b13d-a0d221c89dd0">

## Checklist


### If this is a change to design:


### If this is a change to code:
